distribution/sc-distribution-rs/src/lib.rs

```diff
@@ -51,23 +51,23 @@ pub trait Distribution: global_op::GlobalOperationModule {
         Ok(())
     }
 
+    #[only_owner]
     #[endpoint(startGlobalOperation)]
     fn start_planning(&self) -> SCResult<()> {
-        only_owner!(self, "Permission denied");
         self.global_op_start()?;
         Ok(())
     }
 
+    #[only_owner]
     #[endpoint(endGlobalOperation)]
     fn end_planning(&self) -> SCResult<()> {
-        only_owner!(self, "Permission denied");
         self.global_op_stop()?;
         Ok(())
     }
```

Comment:
Can return the SCResult from the underlying function directly instead of unwrapping the result, and then manually returning an Ok(())